src/vivarium_cluster_tools/cli_tools.py

```diff
@@ -11,10 +10,10 @@
 
 import click
 
-Decorator = Callable[[Callable], Callable]
+Decorator = Callable[[Callable[..., object]], Callable[..., object]]
 
 
-def with_verbose_and_pdb(func: Callable) -> Callable:
+def with_verbose_and_pdb(func: Callable[..., object]) -> Callable[..., object]:
```

Comment:
Doesn't this destroy the type hinting of the wrapped function? Shouldn't we use a TypeVar here to show that the signature of the wrapped function hasn't changed?

Reply:
I guess it does, yeah - this is just the Callable equivalent of using Any (... for inputs and object for the return). I'm still not very comfortable using TypeVars but I'll give it a shot